sdk/storage/storage-internal-avro/package.json

```diff
@@ -3,7 +3,7 @@
   "sideEffect": false,
   "private": true,
   "author": "Microsoft Corporation",
-  "version": "1.0.0-preview.1",
+  "version": "1.0.0",
```

Comment:
0.0.1?

Reply:
This is internal that won't be released. So won't maintain its version.